pontoon/base/models.py

```diff
@@ -1272,46 +1272,49 @@ def get_translation(self, plural_form=None):
             }
 
     @classmethod
-    def for_project_locale(self, project, locale, paths=None, filter_type=None,
-        search=None, exclude=None):
+    def for_project_locale(self, project, locale, paths=None, by_type=None,
+        search=None, exclude=None, by_author=None, by_time=None):
         """Get project entities with locale translations."""
-        if filter_type and filter_type != 'all':
-            if filter_type == 'missing':
-                entities = self.objects.missing(locale)
+        entities = Entity.objects.all()
+        
+        if by_type or by_author or by_time:
```

Comment:
After re-reading this code second time, I may be wrong but could we remove this condition? This makes this part of code more than it's necessary, also you check these values twice.